testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketGatheringWriteTest.java

```diff
@@ -99,6 +100,7 @@ public void testGatheringWriteWithComposite(ServerBootstrap sb, Bootstrap cb) th
 
     // Test for https://github.com/netty/netty/issues/2647
     @Test
+    @Ignore
```

Comment:
Is this a debugging left-over, or are we ignoring this test for now?

Reply:
yep ... sorry let me remove this one.